tests/unit/test_noise_form.py

```diff
@@ -1,20 +1,25 @@
 import random
+from enum import Enum
 from string import ascii_lowercase
+from typing import NamedTuple
 
+import numpy as np
 import pandas as pd
 import pytest
 from vivarium.config_tree import ConfigTree
 
-from pseudopeople.entities import Form
+from pseudopeople.entity_types import ColumnNoiseType, RowNoiseType
 from pseudopeople.interface import generate_decennial_census
-from pseudopeople.noise import NOISE_TYPES, noise_form
+from pseudopeople.noise import noise_form
+from pseudopeople.noise_entities import NOISE_TYPES
+from pseudopeople.schema_entities import Form
 
 
 @pytest.fixture(scope="module")
 def dummy_data():
     """Create a two-column dummy dataset"""
     random.seed(0)
-    num_rows = 1_000_000
+    num_rows = 100_000
```

Comment:
Did you discuss 100k being acceptble or did you just forget to change it back after testing?